apps/devportal/data/markdown/pages/learn/getting-started/xm-cloud/tutorials/setup-xm-cloud.md

```diff
@@ -0,0 +1,183 @@
+ 
+
+## Working locally  
+
+Navigate to your GitHub account and download the repository that was cloned earlier by the Sitecore XM Cloud Deploy App. 
+![GitHub - Clone Repository](https://sitecorecontenthub.stylelabs.cloud/api/public/content/239fa0b1ce314ca088bc830ee7a699e8?v=ba0d1b2e)
+
+### Download Repository
+
+1. In your repositories overview, find the newly created xm-cloud-tutorial-series repo.  
+2. Download the repository to your local file system using the GitHub CLI. Now, open the newly created folder in VS Code, or your text editor of choice.  
+3. As per the README, run the init script that was delivered with your source code repository passing the path to your XM Cloud license file, and the admin password of your XM Cloud instance (“b”, in this case). The init script initializes the .env file with everything necessary.  
+![In Command Line Tool - Run init script](https://sitecorecontenthub.stylelabs.cloud/api/public/content/8cb688885bc74cf9a3b44468dd58ff57?v=0ca77d8f) 
+ 
+ 
+*A note on passwords:  
+You can use any password and will of course have to use a secure password when dealing with client projects.  But for now we use “b”, for tradition’s sake. * 
+
+### Connect the App to the XM Cloud Instance  
+
+As you may remember from the introduction, you need to use a GraphQL endpoint to query the content coming from XM Cloud. To connect a few things need to be configured upfront.
+
+1. Navigate to the .env file within your solution under src\sxastarter, and copy it.  
+
+2. Name it .env.local. We do this so that the file is ignored by git by default, so that you do not pass valuable configuration secrets into any repositories.  
+ 
+You can connect using the delivery endpoint of Edge or to the preview endpoint of your CM instance.   
+![XM Cloud Architecture - Connect to Preview Endpoint](https://sitecorecontenthub.stylelabs.cloud/api/public/content/e703be66170a49268beb6564c4c34df7?v=5184aace) 
+
+Any app showing content to the public must connect to the Edge Delivery Endpoint that is scaled and geographically distributed to serve the right performance. Only published content will be available here. For development purposes, you will connect to the preview endpoint as you do not need to care about publishing at this point, so created content is instantly available. This will simplify the development workflow.    
+ 
+
+Now, you need to set the API key within .env.local. The API key should be already available within your XM Cloud instance.  
+ 
+3. Now you need to set the Context ID in the .env.local file. Therefore  
+ * Go to the XM Cloud deploy app 
+ * Click on the name of the project you created e.g. XM Cloud Tutorial Series 
+ * Copy the Context ID (Preview) 	 
+![Get Context ID from XM Cloud Deploy](https://sitecorecontenthub.stylelabs.cloud/api/public/content/8ecc65168a504cc188ee84cd8951244b?v=a04033a5) 
+4. Add the Context ID to the `.env.local` file 
```

Comment:
This should specify the name of the environment variable we are replacing, should not rely only on the screenshot.

Reply:
Resolved in latest commit